stingray/io.py

```diff
@@ -174,6 +174,211 @@ def _get_detector_id(lctable):
     return None
 
 
+def lcurve_from_fits(
+    fits_file,
+    gtistring="GTI",
+    timecolumn="TIME",
+    ratecolumn=None,
+    ratehdu=1,
+    fracexp_limit=0.9,
+    outfile=None,
+    noclobber=False,
+    outdir=None,
+):
+    """
+    Load a lightcurve from a fits file and save it in HENDRICS format.
```

Comment:
Can you be more precise what you mean by "HENDRICS format"? Does that mean the user needs to have HENDRICS installed for this to work? Is this a format that only HENDRICS can read?

Reply:
Argh, this was a leftover from the HENDRICS docs